src/scss/config/color/_content.scss

```diff
@@ -9,3 +9,10 @@ $shadow: hsla(var(--ccs-h--neutral), var(--ccs-s--fg6), var(--ccs-l--fg6), 25%);
 $action: var(--ccs-prime--fg3);
 $active: $highlight;
 $icon: var(--ccs-special--fg1);
+$scroll-panel-bg: hsl(
+  var(--ccs-h--prime),
+  var(--ccs-s--bg4),
+  var(--ccs-l--bg5)
+);
+$scrollbar-track: var(--ccs-prime--bg4-fade);
+$scrollbar-thumb: var(--ccs-prime--bg6-fade);
```

Comment:
Wow it's hard to inspect what these actually do. As soon as you open the inspector, all the search results disappear? I don't know if that's good or bad in terms of the actual UX. But I'm really curious: is Algolia actually hijacking scroll somehow, or are they using some experimental css to style the default scrollbars, or something else?

Reply:
@mirisuzanne you can test it easier with this pro tip:

> @stacyk While working on this, you can add debug: true to the options passed into algoliasearchNetlify in src/js/index.js -- that will allow the dropdown to stay open instead of auto-closing on blur, so you can inspect the markup etc. Let me know if you have any other questions, or if it would be easier to look at this together on a call.

https://github.com/oddbird/oddleventy/pull/204#issuecomment-1100216083

Reply:
thanks, looks good!